tutorials/roofit/rf711_lagrangianmorph.py

```diff
@@ -0,0 +1,178 @@
+config.folderNames = samplelist
+config.couplings.add(cHq3)
+config.couplings.add(kSM)
+
+
+# Create morphing function
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+morphfunc = ROOT.RooLagrangianMorphFunc("morphfunc", "morphed dist. of pTV", config)
+
+# Get morphed distribution at cHq3 = 0.01, 0.5
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+morphfunc.setParameter("cHq3", 0.01)
+morph_hist_0p01 = morphfunc.createTH1("morph_cHq3=0.01")
+morphfunc.setParameter("cHq3", 0.25)
+morph_hist_0p25 = morphfunc.createTH1("morph_cHq3=0.25")
+morphfunc.setParameter("cHq3", 0.5)
+morph_hist_0p5 = morphfunc.createTH1("morph_cHq3=0.5")
+morph_datahist_0p01 = ROOT.RooDataHist("morph_dh_cHq3=0.01", "", ROOT.RooArgList(obsvar), morph_hist_0p01)
+morph_datahist_0p25 = ROOT.RooDataHist("morph_dh_cHq3=0.25", "", ROOT.RooArgList(obsvar), morph_hist_0p25)
+morph_datahist_0p5 = ROOT.RooDataHist("morph_dh_cHq3=0.5", "", ROOT.RooArgList(obsvar), morph_hist_0p5)
+
+# Extract input templates for plotting
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+input_hists = {sample: ROOT.TFile.Open(infilename).Get(sample).FindObject(observablename) for sample in samplelist}
+input_datahists = {
+    sample: ROOT.RooDataHist("dh_" + sample, "dh_" + sample, ROOT.RooArgList(obsvar), input_hists[sample])
+    for sample in samplelist
+}
+
+# Plot input templates
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+frame0 = obsvar.frame(ROOT.RooFit.Title("Input templates for p_{T}^{V}"))
+for sample, color in zip(samplelist, [ROOT.kBlack, ROOT.kRed, ROOT.kBlue]):
+    input_datahists[sample].plotOn(frame0, Name=sample, LineColor=color, MarkerColor=color, MarkerSize=1)
+
+# Plot morphed templates for cHq3=0.01,0.25,0.5
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+frame1 = obsvar.frame(ROOT.RooFit.Title("Morphed templates for selected values"))
+morph_datahist_0p01.plotOn(
+    frame1,
+    Name="morph_dh_cHq3=0.01",
+    DrawOption="C",
+    LineColor=ROOT.kGreen,
+    DataError=getattr(ROOT.RooAbsData, "None"),
+    XErrorSize=0,
+)
+morph_datahist_0p25.plotOn(
+    frame1,
+    Name="morph_dh_cHq3=0.25",
+    DrawOption="C",
+    LineColor=ROOT.kGreen + 1,
+    DataError=getattr(ROOT.RooAbsData, "None"),
+    XErrorSize=0,
+)
+morph_datahist_0p5.plotOn(
+    frame1,
+    Name="morph_dh_cHq3=0.5",
+    DrawOption="C",
+    LineColor=ROOT.kGreen + 2,
+    DataError=getattr(ROOT.RooAbsData, "None"),
+    XErrorSize=0,
+)
+
+# Create wrapped pdf to generate 2D dataset of cHq3 as a function of pTV
+# -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -- -
+
+model = ROOT.RooWrapperPdf("wrap_pdf", "wrap_pdf", morphfunc)
+data = model.generate(ROOT.RooArgSet(cHq3, obsvar), 1000000)
```

Comment:
```suggestion
data = model.generate({cHq3, obsvar}, 1000000)
```
With the new Pythonizations, Python sets can be used in place of RooArgSets for less verbose code.